src/app/services/oauthRequest.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+
+/**
+ * @ngdoc function
+ * @name flightNodeDemo.services:oauthRequest
+ * @description
+ * # oauthRequest
+ * Service for making HTTP request using a JSON Web Token (jwt).
+ */
+angular.module('oauthRequest', [])
+	.factory('oauthRequest', ['$cookies', '$http', function ($cookies, $http) {
+
+		var TOKEN_KEY = 'org.flightnode.jwt';
+
+		return {
+			getToken: function() {
+				return $cookies.get(TOKEN_KEY);
```

Comment:
I am a little worried that there is a possibility of an attacker clobbering `$cookies`, but I need to do more research and possibly a proof of concept of the attack. I think that hijacking would require a successful XSS attack as well, so there may be mitigating controls here.